pulp_smash/pulp3/utils.py

```diff
@@ -159,16 +159,16 @@ def inner(repo, version_href=None):
         repo_version = client.get(version_href)
 
         content = defaultdict(list)
-        for content_type, content_url in repo_version[content_field].items():
-            typed_content = client.get(content_url)
+        for content_type, content_dict in repo_version['content_summary'][content_field].items():
+            typed_content = client.get(content_dict['href'])
             content[content_type] = typed_content
         return content
     return inner
 
 
-get_content = _build_content_fetcher('content_hrefs')  # pylint:disable=invalid-name
-get_added_content = _build_content_fetcher('content_added_hrefs')  # pylint:disable=invalid-name
-get_removed_content = _build_content_fetcher('content_removed_hrefs')  # pylint:disable=invalid-name
```

Comment:
You will want to change lines 174-205, also.  We should probably re-think the API to better suit the new layout, but for the time being let's just keep backwards compatibility.

Reply:
fixed, ty